plugins/scaffolder-backend/src/scaffolder/actions/builtin/catalog/fetch.ts

```diff
@@ -106,7 +116,13 @@ export function createFetchCatalogEntityAction(options: {
       }),
     },
     async handler(ctx) {
-      const { entityRef, entityRefs, optional } = ctx.input;
+      const {
+        entityRef,
+        entityRefs,
+        optional,
+        defaultKind = 'User',
```

Comment:
I don't think we should default these at all if they're not provided. `parseEntityRef` can take `undefined` as `defaultKind` and `defaultNamespace` too. It's unexpected that now we would start to receive entities for things that were previously broken I would say.